src/pages/content/grammar.mdx

```diff
@@ -1,12 +1,146 @@
 ---
 title: Grammar
-description: >
-  Lorem ipsum dolor sit amet, consectetur adipiscing
-  elit, sed do eiusmod tempor incididunt ut labore et
-  dolore magna aliqua.
 ---
 
-Coming soon
+## Acronyms and abbreviations
+
+Acronyms are like inside jokes. If you aren’t in the know they don’t make sense. Use acronyms to
+save space, but not at the risk of confusing people. Spell out an acronym on first use and include
+it in parentheses. ​
+
+_A service level agreement (SLA) is a contract between you and your customers._
+
+### Bold and italics
+
+Use bold to refer to UI elements. Avoid italics or quotation marks. Using bold is a bold statement.
+A little bit goes a long way, so use bold sparingly. ​ Don't get too bold in notifications. Only use
+bold for headings, and not in the description.
+
+<BestPractice>
+  <Do>
+    <p>
+      Click on <b>Settings</b> in Admin Center
+    </p>
+  </Do>
+  <Dont>
+    <p>Click on “Settings” in Admin Center</p>
+  </Dont>
+</BestPractice>
```

Comment:
@rossmoody Noticed that in these examples, the example of what to do/not do is actually in the body of the component, contrasted with our component examples where the example image is above the text content. 

Are you cool with the deviation?

Reply:
I'm ok with it. Good opportunity to rethink the component in the future, though.